### PR TITLE
fix: change warnings to info, increase time between messages

### PR DESCRIPTION
**Pull request by zerbitx**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind enhancement

resolves #ENG-4817

### CODE REVIEW

File: pkg/platform/helper.go [hunk 1]
```diff
@@ -1007,7 +1007,7 @@ func WaitForVCluster(ctx context.Context, client Client, clusterName, spaceName,
 	}
 
 	now := time.Now()
-	nextMessage := now.Add(waitDuration)
+	nextMessage := now.Add(defaultPollInterval)
```
Comment (lizardruss) [inline pkg/platform/helper.go:1010]: Is it intentional to use this instead of `pollInterval` here?

File: pkg/platform/helper.go [hunk 2]
```diff
@@ -1020,7 +1020,7 @@ func WaitForVCluster(ctx context.Context, client Client, clusterName, spaceName,
 				log.Info("Waiting for virtual cluster to be available...")
 			}
 
-			nextMessage = time.Now().Add(waitDuration)
+			nextMessage = time.Now().Add(defaultPollInterval)
```
Comment (lizardruss) [inline pkg/platform/helper.go:1023]: Same question as above

**Review by lizardruss — CHANGES_REQUESTED** [review]
Possible change to make the usage of `defaultPollInterval` consistent

File: pkg/platform/helper.go (hunk 1 above)
Comment (zerbitx) [reply pkg/platform/helper.go]: Ah, no.   That function is actually not used anywhere. I'll remove it.

**Review by FabianKramm — APPROVED** [review]

**Pull request merged**